packages/shared/src/broker.ts

```diff
@@ -132,53 +106,32 @@ const makeRpcRequest = async <
   method: T,
   ...params: z.input<(typeof requestValidators)[T]>
 ): Promise<z.output<(typeof responseValidators)[T]>> => {
-  const body = await fetchWithMaxSize(url, {
-    signal: AbortSignal.timeout(15000),
-    maxSize: 1024, // 1kb
-    method: 'POST',
-    headers: { 'Content-Type': 'application/json' },
-    body: JSON.stringify({
-      jsonrpc: '2.0',
-      id: 1,
-      method: `broker_${method}`,
-      params: requestValidators[method].parse(params),
-    }),
+  const res = await axios.post(url.toString(), {
+    jsonrpc: '2.0',
+    id: 1,
+    method: `broker_${method}`,
+    params: requestValidators[method].parse(params),
   });
 
-  if (body.error) {
-    throw new Error(`request failed with error ${body.error}`);
-  }
-
-  return responseValidators[method].parse(body.result);
+  return responseValidators[method].parse(res.data.result);
 };
 
 export async function requestSwapDepositAddress(
   swapRequest: NewSwapRequest,
-  opts?: { url: string; commissionBps: number },
+  opts: { url: string; commissionBps: number },
```

Comment:
These changes seem not to be backward compatible, is it ok to do them?

Reply:
Yeah, the `shared` package is an internal library. These functions aren't exposed directly to the outside world.